libnestutil/config.h.in

```diff
@@ -129,7 +129,7 @@
 /* "Define if NAN is available" */
 #cmakedefine HAVE_NAN 1
 
-/* "Define if std::nan(NULL) is available" */
+/* "Define if std::nan(nullptr) is available" */
```

Comment:
```suggestion
/* "Define if std::nan(const char*) is available" */
```
(see https://en.cppreference.com/w/cpp/numeric/math/nan) or just
```suggestion
/* "Define if std::nan() is available" */
```

As an aside: We now require C++11, and this function is guaranteed to be in C++11. Therefore, do we need this test? And which other tests and pertaining conditionals could we remove? That should be a follow-up issue.

Reply:
I created an issue #2508, as I agree that we should simply remove this check now, as well as some others